infra/datasource/core/src/test/java/org/apache/shardingsphere/infra/datasource/props/DataSourcePropertiesCreatorTest.java

```diff
@@ -81,4 +102,26 @@ private Map<String, Object> createProperties() {
         result.put("maximumPoolSize", "-1");
         return result;
     }
+    
+    private Map<String, Object> createStandardProperties() {
+        Map<String, Object> result = new LinkedHashMap<>(6, 1F);
+        // This is to test the case where the property type is mismatched.
```

Comment:
Hi @zhangfengcdt 
Thanks for your PR!
According to [Code of Conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/), comments can only be used for Javadoc, todo and fixme, so please remove this line of comment

Reply:
Sure, comment removed.